doc/nrf/glossary.rst

```diff
@@ -414,21 +457,30 @@ Glossary
    Receive Data (RXD)
       A signal line in a serial interface that receives data from another device.
 
+   Received Signal Strength Indication (RSSI)
+      An indication of the power of a received radio signal.
+
+   Remote Procedure Call (RPC)
+      A form of inter-process communication that allows for calling a function on remote processors from a local processor, both synchronously and asynchronously.
```

Comment:
@carlescufi @mbolivar-nordic Can you check if this definition is actually correct in the NCS context?